src/datagrids/IndexDataTable.tsx

```diff
@@ -38,26 +40,54 @@ function IndexDataTable() {
   }, []);
 
   const [rowData, setRowData] = useState([]);
+  const [pageSize, setPageSize] = useState(10);
+  const [pageIndex, setPageIndex] = useState(0);
 
   // @ts-ignore
+  useEffect(() => {
+    //change page size
+  }, [pageSize]);
+  useEffect(() => {
+    //change page
+  }, [pageIndex]);
+
+  const onNextPage = () => {
+    //next page
+    const index = pageIndex + 1;
+    const pageMax = Math.floor(rowData.length/pageSize)
+    index > pageMax ? setPageIndex(pageMax) : setPageIndex(index);
+    
+  }
+  
+  const onLastPage = () => {
+    //previous page
+    const index = pageIndex - 1;
+    index < 0 ? setPageIndex(0) : setPageIndex(index);
+  }
+  
+  const onPageSizeChanged = (e) => {
+    // page size
+    setPageSize(e.target.value);
+  }
+
+
   const onCellClicked = (params) => {
     if (params.column.colId === 'delete') {
       params.api.applyTransaction({
         remove: [params.node.data],
       });
     }
   };
-
   const [columnDefs] = useState([
-    { field: 'indexName', headerName: 'Index Name', width: 150 },
-    { field: 'status', headerName: 'Status', cellRenderer: StatusColumnRenderer },
-    { field: 'reqSec', headerName: 'Requests/Sec' },
-    { field: 'residentRatio', headerName: 'Resident Ratio' },
-    { field: 'items', headerName: 'Items', width: 100 },
-    { field: 'dataSize', headerName: 'Data Size' },
-    { field: 'bucket', headerName: 'Bucket' },
-    { field: 'scope', headerName: 'Scope' },
-    { field: 'collection', headerName: 'Collection' },
+    { field: 'indexName', headerName: 'INDEX NAME', width: 150, cellStyle: {fontWeight: 'bold'}},
+    { field: 'status', headerName: 'STATUS', cellRenderer: StatusColumnRenderer },
+    { field: 'reqSec', headerName: 'REQUEST/SEC', width: 150},
+    { field: 'residentRatio', headerName: 'RESIDENT RATIO', width: 150},
```

Comment:
I think we can expect that the headers will **always** be uppercased and therefore this should be included in the theme, rather than hard coded